.github/workflows/validate_and_publish_release.yml

```diff
@@ -25,6 +25,12 @@ jobs:
         with:
           release_id: ${{ github.event.inputs.release_id }}
 
+      - name: Checking out repository
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.WRITE_ACCESS_TOKEN }}
+          ref: "${{ env.tag_name }}"
+
       # Future different releases should publish to different branches
```

Comment:
I'm not sure that's totally true. We probably don't want to create a new persistent branch for each release. That's what tags are for :-)

Reply:
Comment removed. The original idea was if the workflow was called by a different release tag, it would still be pushed to the "latest-snapshot" branch. If that's not a concern, I can leave it out.